packages/suite/src/hooks/wallet/sign-verify/useSignVerifyForm.ts

```diff
@@ -1,10 +1,10 @@
 import { useEffect } from 'react';
 import { useForm, useController } from 'react-hook-form';
-import { useTranslation } from '@suite-hooks';
-import { isHex } from '@wallet-utils/ethUtils';
-import { isAscii } from '@trezor/utils';
+import yup, { type ValidationSchema } from '@trezor/validation';
```

Comment:
no need to use `type` here

hint: use `type` in imports only when TS complaining about it

Reply:
Oh, ok. Tomáš K said you agreed to use `type` whenever a type is imported, I might have misunderstood :)

Reply:
I will clarify this again on next code improvements, but if you remove it and TS doesn't complain then it is ok